src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NSDbSinkWriter.scala

```diff
@@ -40,7 +40,8 @@ class NSDbSinkWriter(connection: NSDB,
                      globalNamespace: Option[String],
                      defaultValue: Option[java.math.BigDecimal],
                      retentionPolicy: Option[Duration],
-                     shardInterval: Option[Duration])
+                     shardInterval: Option[Duration],
+                     semanticDelivery: String)
```

Comment:
why `semanticDelivery` is not an `Option` like other parameters?

Reply:
Semantic delivery prop is optional but it has to have a default value. So I went for without option. It's a good point...what does @saveveltri think about it?

Reply:
good point. We can keep the structure uniform and change `semanticDelivery` into an option. Later in the execution flow all the other options are evaluated and a default value is used if some of them is none. 
It's only a metter of code claness to evaluate all the options in the same place

Reply:
agree...done!